core/templates/dev/head/dashboard/DashboardServices.js

```diff
@@ -0,0 +1,76 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Standalone services for the creator dashboard page.
+ */
+
+oppia.constant('EXPLORATIONS_SORT_BY_KEYS', {
```

Comment:
As discussed in meeting: Constants will be kept and use to prevent magic strings. Though I had used the constants incorrectly before, which has been corrected in the latest commit.